src/renderer/components/+preferences/preferences.tsx

```diff
@@ -48,26 +46,40 @@ import { Kubernetes } from "./kubernetes";
 import { Editor } from "./editor";
 import { LensProxy } from "./proxy";
 import { Telemetry } from "./telemetry";
-import { Extensions } from "./extensions";
 import { sentryDsn } from "../../../common/vars";
+import { withInjectables } from "@ogre-tools/injectable-react";
+import type { InstalledExtension } from "../../../extensions/extension-discovery";
+import userExtensionsInjectable from "../+extensions/user-extensions/user-extensions.injectable";
+import { ExtensionSettingsPage } from "./extension-settings-page";
+import { Icon } from "../icon";
+import { uniqBy } from "lodash";
+
+interface Dependencies {
```

Comment:
Why do we call it `Dependencies`? I'd better to stick to `Props` name for component props interface.

Reply:
For example see https://github.com/lensapp/lens/pull/4653/files#diff-9439f9090e19e6ccbf3899490a1995b5bf2a63efd852dc6df1c76c1ce3aae53f from @jansav's PR

Reply:
I'm fine with  `interface Props extends Dependencies { ... }`

Reply:
But we don't want that in the documentation. The docs should not contain the dependencies of the components